src/Raven.Server/Documents/Subscriptions/SubscriptionConnectionBase.cs

```diff
@@ -51,6 +52,7 @@ public abstract class SubscriptionConnectionBase<TIncludesCommand> : ISubscripti
         internal SubscriptionWorkerOptions _options;
         internal readonly Logger _logger;
 
+        public string LastSentChangeVectorInThisConnection;
```

Comment:
you are passing a lot of state via instance fields, please check if this is really necessary

Reply:
I just moved it from `SubscriptionConnection` to base class (so I can use it in orchestrator).
In general AFAIK we have this field, since in concurrent subscription it saves the cv for particular connection, and the field in state save the merged one.